runtime/shibuya/src/xcm_config.rs

```diff
@@ -241,7 +241,9 @@ impl pallet_xcm::Config for Runtime {
     type TrustedLockers = ();
     type SovereignAccountOf = LocationToAccountId;
     type MaxLockers = ConstU32<0>;
-    type WeightInfo = pallet_xcm::weights::SubstrateWeight<Runtime>;
+    type WeightInfo = pallet_xcm::TestWeightInfo;
```

Comment:
What should we do about the weights? MB was using these weights up until a month ago but now they have benchmarked it.

Reply:
Benchmarked weights added.